lib/sles4sap_publiccloud_lib.pm

```diff
@@ -0,0 +1,148 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Library used for SLES4SAP publicccloud deployment and tests
+
+use base 'publiccloud::basetest';
+package sles4sap_publiccloud_lib;
+use strict;
+use warnings FATAL => 'all';
+use testapi;
+use List::MoreUtils qw(uniq);
+use Exporter 'import';
+use Carp qw(croak);
+
+our @EXPORT = qw(
+  run_cmd
+  get_promoted_hostname
+  get_hana_topology
+  wait_for_sync
+);
+
+# Global variables
+our $crm_mon_cmd = 'crm_mon -R -r -n -N -1';
```

Comment:
You can 

```
use hacluster '$crm_mon_cmd';
```

Reply:
+1

Reply:
Yep, good idea. Changed with last push.
@alvarocarvajald I changed the filename as well to lib/sles4sap_publiccloud.pm :)